package.json

```diff
@@ -24,14 +24,11 @@
     "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
-    "lint:fix": "yarn lint:js --fix",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
     "start:dev": "ember server --proxy https://conduit.productionready.io",
-    "test": "npm-run-all lint:* test:*",
```

Comment:
In my experience, it's better to separate linting and testing for CI. I updated the default `test` script that Ember CLI provides, then removed `test:s` (I don't think it's needed when `yarn test --server` suffices).

Reply:
So these changes were brought in from a recent ember-cli upgrade ([see this long discussion](https://github.com/ember-cli/ember-cli/pull/9009) for context on why these script changes were made).

As we are trying to have this app mirror an out-of-the-box Ember app as much as possible maybe we should use `yarn test:ember` and `yarn lint` separately in CI rather than remove the other scripts?

Reply:
Yep, that sounds good with me. I'll undo this change and the `nolint` change.